en/advanced/px4_metadata.md

```diff
@@ -1,15 +1,17 @@
-# PX4 Metadata: Translation & Publishing (Params, Events)
+# PX4 Metadata
 
 PX4 uses and generates data that has associated human- and machine- readable metadata:
 
 - [Parameters](../advanced_config/parameters.md) configure PX4 behaviour.
   - A parameter is represented by an ID string that maps to a value stored in PX4.
   - The associated metadata includes a description of the setting, its possible values, information about how the value might be presented (say for bitmasks).
 - [Events](../concept/events_interface.md) provide notification of events, such as reasons for a failsafe, low battery warnings, end of calibration, and so on.
-  - An event is represented by an id, and is sent with a log level, some message, and arguments.
-  - The associated metadata includes a description of the event and the arguments.
+  - An event is represented by an id, and is sent with a log level and arguments.
+  - The associated metadata includes a message, a description and a list of arguments (including their type) of each event.
+- [Actuators](../config/actuators.md) metadata contains information about supported vehicle geometries, a list of output drivers, and how to configure them.
+  QGroundControl uses that information to dynamically build a configuration UI.
 
```

Comment:
This is not quite symmetric with the other headings above. I'll add this, but if you think it is wrong we can revert.

```suggestion
- [Actuators](../config/actuators.md) configuration customizes the specific geometry of the vehicle, assigns actuators and motors to flight controller outputs, and tests the actuator and motor response.
  - The metadata contains information about supported vehicle geometries, a list of output drivers, and how to configure them.
  - _QGroundControl_ uses that information to dynamically build a configuration UI.

```